autotest/ogr/ogr_flatgeobuf.py

```diff
@@ -247,3 +247,8 @@ def test_ogr_flatgeobuf_9():
         rc = verify_flatgeobuf_copy(test[0], test[1], test[2])
         assert rc, ('Verification of copy of ' + test[0] + '.shp failed')
 
+def test_ogr_flatgeobuf_ossfuzz_17603():
+    fgb_ds = ogr.Open('data/fgb/clusterfuzz-testcase-minimized-fgb_fuzzer-5725251593306112.fgb')
```

Comment:
I've very rarely included myself a ossfuzz generated test case in autotest, because they are usually corrupted in several ways, and a later hardening fix might make one test case completely irrelevant (or completely rejected) for what it used to be tested to. So when possible, I try to corrupt a valid dataset in a unit way, just testing one potential defect. But this is a lot of work, potentially requiring to do coding for binary formats, so I tend not doing it.
Just wanting to share my experience/thoughts about that. I'm fine with you letting this at it (that's what oss-fuzz actually recommands). Just that at some later point it might no longer test-stress the code you fixed.

Reply:
Ok yes, I can see it becoming worthless to regress test the fuzz reproductions and it might be difficult to know when that happens so I'll opt for not having it, removed now.